### PR TITLE
feat: add expiry notices to connection cards

### PR DESCRIPTION
**Pull request by rolznz**
image

### CODE REVIEW

File: frontend/src/components/connections/AppCardNotice.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                    &nbsp; Expired
+                  </Button>
+                </Link>
+              </TooltipTrigger>
+              <TooltipContent>
+                Expired {dayjs(app.expiresAt).fromNow()}
+              </TooltipContent>
+            </Tooltip>
+          </TooltipProvider>
+        ) : new Date(app.expiresAt).getTime() - ONE_WEEK_IN_SECONDS <
```
Comment (bumi) [inline frontend/src/components/connections/AppCardNotice.tsx:39]: you could use dayjs as we have it and use it.

**Review by bumi — COMMENTED** [review]
I'd use dayjs also for the calculations

**Comment by reneaaron** [comment]
I've addressed @bumi's feedback and tried to optimize the UI a bit. 

Great refactoring @rolznz, thanks for splitting up the components! :+1:

File: frontend/src/components/connections/AppCardConnectionInfo.tsx [hunk 2]
```diff
@@ -42,30 +30,19 @@ export function AppCardConnectionInfo({
...
+            <div>
+              {connection.maxAmount && (
+                <>{formatAmount(connection.maxAmount * 1000)} sats</>
+              )}
             </div>
-          </div>
-          {connection.lastEventAt && (
-            <div className="flex flex-row justify-between text-xs items-center">
-              <div className="flex flex-row justify-between">
-                <div>Last used:&nbsp;</div>
```
Comment (rolznz) [inline frontend/src/components/connections/AppCardConnectionInfo.tsx]: `Last used` was removed from the card, is this intentional?

Comment (rolznz) [reply frontend/src/components/connections/AppCardConnectionInfo.tsx]: @reneaaron I think it's useful information to have

Comment (reneaaron) [reply frontend/src/components/connections/AppCardConnectionInfo.tsx]: @rolznz It definitely is, however the cards are already quite clogged with data. I wonder if it's really worth showing this info on the overview (we show it on the detail page anyway).

Comment (rolznz) [reply frontend/src/components/connections/AppCardConnectionInfo.tsx]: I agree the card is a bit cluttered, maybe we can find a better solution. But having to manually click all the cards to open them to see which ones were used recently is not so great in my opinion

**Pull request merged**